dev-requirements.txt

```diff
@@ -1,8 +1,9 @@
 funcy>=1.10
 Keras==2.2.4
 numpy>=1.10.0
-pylint
+pylint>=2.4.0
```

Comment:
I think you might want `pylint>=2.4.0,<3` instead. This is so that if `pylint==3.0.0` comes out with breaking changes, we don't accidentally pick it up.